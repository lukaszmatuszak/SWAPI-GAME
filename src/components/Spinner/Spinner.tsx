import React from 'react';
import { CircularProgress, Box } from '@material-ui/core';

const Spinner: React.FC = () => (
    <Box
        display="flex"
        height="calc(100vh - 48px)"
        alignItems="center"
        justifyContent="center"
    >
        <CircularProgress size={70} />
    </Box>
);

export default Spinner;
