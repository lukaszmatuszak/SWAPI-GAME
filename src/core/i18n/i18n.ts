import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';
import LanguageDetector from 'i18next-browser-languagedetector';

import locales from './locales/index';

i18n.use(initReactI18next)
    .use(LanguageDetector)
    .init({
        ns: ['Common'],
        defaultNS: 'Common',
        resources: locales,
        fallbackLng: 'en',
        interpolation: {
            escapeValue: false,
        },
    });
