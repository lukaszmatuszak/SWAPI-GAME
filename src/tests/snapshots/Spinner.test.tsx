import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Spinner from '../../components/Spinner/Spinner';
import '../../core/i18n/i18n';

describe('Spinner Snapshot - ', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Spinner />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
