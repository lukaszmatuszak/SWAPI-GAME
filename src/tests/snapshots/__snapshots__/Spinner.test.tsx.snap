// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Snapshot -  renders correctly 1`] = `
<Styled(MuiBox)
  alignItems="center"
  display="flex"
  height="calc(100vh - 48px)"
  justifyContent="center"
>
  <WithStyles(ForwardRef(CircularProgress))
    size={70}
  />
</Styled(MuiBox)>
`;
